documentation/configuring_bolt.md

```diff
@@ -1,59 +1,190 @@
 # Configuring Bolt
 
 You can configure Bolt's options and features at a project level, a user level,
-or a system-wide level. 
+or a system-wide level.
 
-Some options and features you can configure include:
-- global options
-- transports 
-- plugins
-- PuppetDB connections
-- log files 
+## Project-level configuration
 
-For a complete list of Bolt settings, see the [configuration
-reference](bolt_configuration_reference.md).
+All configurable options in Bolt can be set at the project level. Configuration
+that should only apply to the current project should be set at this level. Most
+of the time, you will only need to set configuration at the project level.
 
-## Configuration files
+Project-level configuration can be set in three locations. Bolt configuration
+can be set in a `bolt-project.yaml` file, inventory configuration can be set in
+an `inventory.yaml` file, and all configuration can be set in a `bolt.yaml`
+file.
 
-To configure Bolt, create a Bolt configuration file named `bolt.yaml` in one of
-the locations listed below. If a configuration file is not located at the
-expected path, or the file is empty, Bolt does not load the file. 
+> **Note:** Project configuration files are loaded from the [Bolt project
+> directory](bolt_project_directories.md). The default project directory is
+> `~/.puppetlabs/bolt/`.
 
-You can place `bolt.yaml` configuration files at the following paths, from
-highest precedence to lowest:
+The preferred method for setting project-level configuration is to use a
+combination of `bolt-project.yaml` and `inventory.yaml` files. This maintains
+a clear distinction between Bolt configuration and inventory configuration.
```

Comment:
Note about deprecation?

Reply:
I like the note below. I guess my only concern right here is that people will read this far, stop reading, create a bolt.yaml because it seems easier (1 file instead of 2), then have it get dropped in a few months.